CMakeLists.txt

```diff
@@ -105,63 +105,32 @@ endif()
 
 # --------------------------------------------------------------------------
 # Install directories
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME)
+include(GNUInstallDirs)
+
 # Build DOCUMENTATION (not in ALL target and only if Doxygen is found)
 option(BUILD_DOC "Build the HTML documentation (with doxygen if available)." OFF)
 
-string(TOLOWER ${PROJECT_NAME} projectname)
-set(OPENJPEG_INSTALL_SUBDIR "${projectname}-${OPENJPEG_VERSION_MAJOR}.${OPENJPEG_VERSION_MINOR}")
-
-if(NOT OPENJPEG_INSTALL_BIN_DIR)
-  set(OPENJPEG_INSTALL_BIN_DIR "bin")
-endif()
-
-if(NOT OPENJPEG_INSTALL_LIB_DIR)
-  set(OPENJPEG_INSTALL_LIB_DIR "lib")
-endif()
-
-if(NOT OPENJPEG_INSTALL_SHARE_DIR)
-  set(OPENJPEG_INSTALL_SHARE_DIR "share")
-endif()
-
-if(NOT OPENJPEG_INSTALL_DATA_DIR)
-  set(OPENJPEG_INSTALL_DATA_DIR "${OPENJPEG_INSTALL_SHARE_DIR}/${OPENJPEG_INSTALL_SUBDIR}")
-endif()
-
-if(NOT OPENJPEG_INSTALL_INCLUDE_DIR)
-  set(OPENJPEG_INSTALL_INCLUDE_DIR "include/${OPENJPEG_INSTALL_SUBDIR}")
-endif()
```

Comment:
This change removes the ability to customize the include dir location in v2.5.1. This affects the vcpkg port. (I wouldn't care, but I learned that Visual Studio users feel less comfortable with versioned include paths if not using cmake.)

Reply:
This would change the prefix (`include` -> `foo`) but not the suffix (`openjpeg-2.5`) because it hardcoded everywhere.

For the vcpkg port I decided to not add a patch (i.e. openjpeg decides), but write two forwarding headers to vcpkg's old location, marked as "legacy".
So probably no further change needed here - but a breaking change nevertheless.

Reply:
@dg0yt If you want to contribute a pull request to fix that, that's a good moment to do because I'll likely have to issue a 2.5.2 because of https://github.com/uclouvain/openjpeg/issues/1514#issuecomment-1966373513